README.md

```diff
@@ -1,17 +1,99 @@
-## MongoDB Atlas CLI
+# MongoDB CLI
 
 ![GO tests](https://github.com/mongodb/mongodb-atlas-cli/workflows/GO%20tests/badge.svg)
```

Comment:
should these links point to the actions of that branch?

Reply:
I don't think these workflows are still there, both badges give a 404
- https://github.com/mongodb/mongodb-atlas-cli/workflows/GO%20tests/badge.svg?branch=mongocli-master
- https://github.com/mongodb/mongodb-atlas-cli/workflows/golangci-lint/badge.svg?branch=mongocli-master